[ERROR]	2025-07-18T15:15:21.188Z	63c02686-e52c-488e-b2c3-b3f1691402ba	Error processing record: 'Bedrock' object has no attribute 'ingest_knowledge_base_documents'
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 112, in lambda_handler
    response = bedrock_client.ingest_knowledge_base_documents(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/lang/lib/python3.13/site-packages/botocore/client.py", line 953, in __getattr__
    raise AttributeError(
        f"'{self.__class__.__name__}' object has no attribute '{item}'"
    )
AttributeError: 'Bedrock' object has no attribute 'ingest_knowledge_base_documents'
